scripts/README.md

```diff
@@ -0,0 +1,17 @@
+# Packaging scripts
+
+This directory contains useful packaging scripts.
+
+## `configure-hypervisor.sh`
+
+This script generates the official set of hypervisor build configuration
```

Comment:
I was thinking in the README - currently we say 'hypervisor', in a rather neutral way, implying to me that it might either understand which hypervisor you are asking for, or support multiple hypervisors. Just changing that to QEMU would work for me, just to be explicit that this is just supporting QEMU and not filtering on the arguments right now.

I'm fine on not filtering right now, the script does exactly what we need it to do for our integrations, just wanted it a bit more explicit that it will spit out QEMU settings, always, right now.